katas/content/oracles/index.md

```diff
@@ -72,10 +72,13 @@ This allows us to define quantum oracles based on their effect on the basis stat
 There are two types of quantum oracles: phase oracles and marking oracles.  Let's take a closer look at them.
 
 ## Phase Oracles
-A phase oracle $U_{phase}$ is an oracle that encodes the value of the classical function $f$ it implements in the *phase* of the qubit state. When provided an input basis state $|\vec{x}\rangle$, it flips the sign of that state if $f(x)=1$:
+
+For a function $f(x): 0,1^n \rightarrow 0,1$, the phase oracle $U_{\text{phase}}$ encodes the the values $f(0)$ and $f(1)$ in the relative phases of basis states $\ket{0}$ and $\ket{1}$, respectively.
```

Comment:
Also should be f, not f(x). Besides 0,1^n doesn't seem right. Should probably be \{0,1\}^n \rightarrow \{0,1\}

Reply:
Fixed.